website_sale_checkout_skip_payment/README.rst

```diff
@@ -76,6 +76,10 @@ Contributors
 * Martin Wilderoth <martin.wilderoth@linserv.se>
 * Alexandre Díaz <alexandre.diaz@tecnativa.com>
 
+* `Studio73 <https://www.studio73.es>`_:
+
+  * Miguel Gandia <miguel@studio73.es>
```

Comment:
This shouldn't be edited here, and it will be lost on merge. Please do it inside `readme` and restore this one to the original one.

Reply:
I fixed it.
Thank you for your review.